### PR TITLE
UIINREACH-43: General INN-Reach Settings UI Enhancements

### PR DESCRIPTION
**Pull request by Dmytro-Melnyshyn**
- added a visual indication of the active setting, 
- added accessibility for each section of settings
- made the 'Save' buttons larger
- removed the constraint on the loading for central servers, 
- added a default option for the contribution criteria form
- fixed a bug related to activating the Save button when selecting the "no selection" option for the contribution criteria form
- added a default option for 'FOLIO to INN-Reach locations'
- changed the name of the local server fields for 'Agency to FOLIO location'
- fixed the save button activation for 'Agency to FOLIO location' (when fields are filled, but no changes)
- fixed the display of location fields if data is available for 'Agency to FOLIO location'
- added an asterisk to 'FolioToInnReachLocations' 
- removed an asterisk from 'AgencyToFolioLocation'
- changed the label of the 'Item contribution options' section to 'Circulation status mapping'
- retrieved central server options
- retrieved common modal text separately

### CODE REVIEW

File: src/settings/index.js [hunk 1]
```diff
@@ -92,14 +94,15 @@ const InnReachSettings = ({
 InnReachSettings.manifest = Object.freeze({
   centralServerRecords: {
     type: 'okapi',
-    path: 'inn-reach/central-servers',
+    path: 'inn-reach/central-servers?limit=1000',
```
Comment (Godlevskyi) [inline src/settings/index.js:97]: Please increase limit to 9999 and move it to central server constants

File: src/settings/routes/CentralServersConfigurationRoute/CentralServersConfigurationCreateRoute.js [hunk 2]
```diff
@@ -108,7 +108,7 @@ const CentralServersConfigurationCreateRoute = ({
 CentralServersConfigurationCreateRoute.manifest = Object.freeze({
   centralServerRecords: {
     type: 'okapi',
-    path: 'inn-reach/central-servers',
+    path: 'inn-reach/central-servers?limit=1000',
```
Comment (Godlevskyi) [inline src/settings/routes/CentralServersConfigurationRoute/CentralServersConfigurationCreateRoute.js:111]: Please use the constant for the limit. Make sure that the limit is applied only for GET

File: src/settings/routes/CentralServersConfigurationRoute/CentralServersConfigurationListRoute.js [hunk 3]
```diff
@@ -89,7 +89,7 @@ const CentralServersConfigurationListRoute = ({
 CentralServersConfigurationListRoute.manifest = Object.freeze({
   centralServerRecords: {
     type: 'okapi',
-    path: 'inn-reach/central-servers',
+    path: 'inn-reach/central-servers?limit=1000',
```
Comment (Godlevskyi) [inline src/settings/routes/CentralServersConfigurationRoute/CentralServersConfigurationListRoute.js:92]: Please use constant for the limit

File: src/settings/routes/ContributionCriteriaRoute/ContributionCriteriaCreateEditRoute.js [hunk 4]
```diff
@@ -184,7 +158,7 @@ const ContributionCriteriaCreateEditRoute = ({
 ContributionCriteriaCreateEditRoute.manifest = Object.freeze({
   centralServerRecords: {
     type: 'okapi',
-    path: 'inn-reach/central-servers',
+    path: 'inn-reach/central-servers?limit=1000',
```
Comment (Godlevskyi) [inline src/settings/routes/ContributionCriteriaRoute/ContributionCriteriaCreateEditRoute.js:161]: Please move the limit to constants

File: src/settings/routes/ContributionOptionsRoute/ContributionOptionsCreateEditRoute.js [hunk 5]
```diff
@@ -242,7 +242,7 @@ const ContributionOptionsCreateEditRoute = ({
 ContributionOptionsCreateEditRoute.manifest = Object.freeze({
   centralServerRecords: {
     type: 'okapi',
-    path: 'inn-reach/central-servers',
+    path: 'inn-reach/central-servers?limit=1000',
```
Comment (Godlevskyi) [inline src/settings/routes/ContributionOptionsRoute/ContributionOptionsCreateEditRoute.js:245]: Please apply the constant value for the limit

File: src/settings/routes/FolioToInnReachLocations/FolioToInnReachLocationsCreateEditRoute.js [hunk 6]
```diff
@@ -336,7 +347,7 @@ FolioToInnReachLocationsCreateEditRoute.manifest = Object.freeze({
   selectedLibraryId: { initialValue: '' },
   centralServerRecords: {
     type: 'okapi',
-    path: 'inn-reach/central-servers',
+    path: 'inn-reach/central-servers?limit=1000',
```
Comment (Godlevskyi) [inline src/settings/routes/FolioToInnReachLocations/FolioToInnReachLocationsCreateEditRoute.js:350]: Same here

File: src/settings/routes/MaterialTypeRoute/MaterialTypeCreateEditRoute.js [hunk 7]
```diff
@@ -192,7 +192,7 @@ const MaterialTypeCreateEditRoute = ({
 MaterialTypeCreateEditRoute.manifest = Object.freeze({
   centralServerRecords: {
     type: 'okapi',
-    path: 'inn-reach/central-servers',
+    path: 'inn-reach/central-servers?limit=1000',
```
Comment (Godlevskyi) [inline src/settings/routes/MaterialTypeRoute/MaterialTypeCreateEditRoute.js:195]: same here

File: src/utils/dropDownMenuOptions.js [hunk 8]
```diff
@@ -0,0 +1,7 @@
+export const getCentralServerOptions = (servers) => {
+  return servers.map(({ id, name }) => ({
+    id,
+    label: name,
+    value: name,
```
Comment (Godlevskyi) [inline src/utils/dropDownMenuOptions.js:5]: Why the value was changed to name?

Comment (Dmytro-Melnyshyn) [reply src/utils/dropDownMenuOptions.js:5]: All mappings besides one had name as value

File: src/settings/routes/AgencyToFolioLocations/AgencyToFolioLocationsCreateEditRoute.js [hunk 9]
```diff
@@ -314,7 +353,7 @@ AgencyToFolioLocationsCreateEditRoute.manifest = Object.freeze({
   selectedServerId: { initialValue: '' },
   centralServerRecords: {
     type: 'okapi',
-    path: 'inn-reach/central-servers',
+    path: 'inn-reach/central-servers?limit=1000',
```
Comment (Godlevskyi) [inline src/settings/routes/AgencyToFolioLocations/AgencyToFolioLocationsCreateEditRoute.js:356]: Same here

**Review by Godlevskyi — APPROVED** [review]

**Pull request merged**